packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -224,6 +224,22 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for Policy Guidelines
+    ///
+
+    // RPC Code: RPC-Policy-V1-03
+    AvoidAdditionalProperties: {
+      description: "The Additional Properties is not allowed except for tags.",
+      severity: "error",
+      message: "{{description}}",
+      resolved: true,
+      formats: [oas2],
+      given: "$..[?(@property == 'additionalProperties')]",
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=34&length=20)

how is tags being ignored in this logic?

Reply:
Added the check